src/transport/transport.h

```diff
@@ -258,86 +331,62 @@ class ConvectionTransport : public ConcentrationTransportBase {
      *
      * Updates CFL time step constrain.
      */
-    void create_transport_matrix_mpi();
-    void create_mass_matrix();
+//    void create_transport_matrix_mpi();
+//    void create_mass_matrix();
 
     void make_transport_partitioning(); //
```

Comment:
Possibly remove, together with el_ds, el_4_loc, and row_4_el. Same for the method get_par_info in TransportOperatorSplitting and in TransportDG. Seems that only el_ds->lsize() is effectively in use and that should be mostly replaced by the size of the DOF handler, or taken directly from the mesh.